src/Library/demos/Row/main.blp

```diff
@@ -0,0 +1,104 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: "Rows";
+  description: "Create user interface lists with ease";
```

Comment:
I think we can remove description here. I can't think of a single sentence that would be a good fit here.